2.9.0 Release notes (#1881)
timescale/docs: timescaledb/overview/release-notes/index.md

</highlight>

## What's new in TimescaleDB 2.9:

* Hierarchical Continuous Aggregates (aka Continuous Aggregate on top of another Continuous Aggregate)
* Improve `time_bucket_gapfill` function to allow specifying the timezone to bucket
* Introduce fixed schedules for background jobs and the ability to check job errors.
* Use `alter_data_node()` to change the data node configuration. This function introduces the option to configure the availability of the data node.

## 2.9.0 (2022-12-15)

This release adds major new features since the 2.8.1 release.
We deem it moderate priority for upgrading.

This release includes these noteworthy features:
* Hierarchical Continuous Aggregates (aka Continuous Aggregate on top of another Continuous Aggregate)
* Improve `time_bucket_gapfill` function to allow specifying the timezone to bucket
* Introduce fixed schedules for background jobs and the ability to check job errors.
* Use `alter_data_node()` to change the data node configuration. This function introduces the option to configure the availability of the data node.

This release also includes several bug fixes.

**Features**

> * #4476 Batch rows on access node for distributed COPY

Reviewer: ```suggestion
* #4476 Batch rows on access node for distributed `COPY`
```

* #4567 Exponentially backoff when out of background workers
* #4650 Show warnings when not following best practices
* #4664 Introduce fixed schedules for background jobs
* #4668 Hierarchical Continuous Aggregates
* #4670 Add timezone support to time_bucket_gapfill
* #4678 Add interface for troubleshooting job failures
* #4718 Add ability to merge chunks while compressing
* #4786 Extend the now() optimization to also apply to CURRENT_TIMESTAMP
* #4820 Support parameterized data node scans in joins
* #4830 Add function to change configuration of a data nodes
* #4966 Handle DML activity when datanode is not available

> * #4971 Add function to drop stale chunks on a datanode

Reviewer: ```suggestion
* #4971 Add function to drop stale chunks on a data node
```

> **Bugfixes**

Reviewer: ```suggestion
**Bug fixes**
```
